client/network/src/service.rs

```diff
@@ -868,6 +894,13 @@ impl Metrics {
 			kbuckets_num_nodes: register(Gauge::new(
 				"sub_libp2p_kbuckets_num_nodes", "Number of nodes in the Kademlia k-buckets"
 			)?, registry)?,
+			listeners_local_addresses: register(Gauge::new(
+				"sub_libp2p_listeners_local_addresses", "Number of local addresses we're listening on"
```

Comment:
```suggestion
				"sub_libp2p_listeners_addresses", "Number of addresses we're listening on"
```
Aren't listener addresses always "local"?

Reply:
It's to differentiate with "external" addresses, that typically contain external IP addresses.